webhook/lib/snt/webhook/api/webhooks.rb

```diff
@@ -0,0 +1,34 @@
+module SNT
+  module Webhook
+    module API
+      class Webhooks < Base
+        class << self
+          # ::SNT::Webhook::API::Webhooks.index(chain_uuid)
+          def index(*args)
```

Comment:
This is all duplication.  Can we use define_method, validate the method names, and call it on the api?